### DISCUSSION TITLE
Running Capacitor App along with a Native App

### DISCUSSION THREAD

**Discussion by zakuru** [ionic-team/capacitor#5995] [Q&A]
Hi all,
First amazing job to all - I am happily surprised by the power of Capacitor
I have a use case where I want to migrate my native app to Capacitor - that would be the north star
But to achieve this migration I would like to migrate progressively, let's stay one view / page at the time
Is it possible to plug an instance of a Capacitor App within a native app ?
More precisely, my first use case is a vanilla web view that I want to convert to a capacitor based app.
Is there any doc or guidance to follow ? 
Thanks

**Comment by jcesarmobile** [comment]
It's easier if you use Ionic Portals for that https://ionic.io/portals

**Reply by zakuru** [reply]
thanks a lot !
